per-seat-subscriptions/server/java/src/main/java/com/stripe/sample/Server.java

```diff
@@ -362,9 +354,19 @@ public static void main(String[] args) {
           )
           .build();
 
-        Subscription subscription = Subscription.create(subCreateParams);
+          Subscription subscription = Subscription.create(subCreateParams);
 
-        return subscription.toJson();
+          return subscription.toJson();
+
+        } catch (StripeException e) {
```

Comment:
For now catching a StripeException instead of just the CardException as that's more inline with what we're doing in the other implementations but down the road I think we may want to show catching different types.